plugins/loading/package.json

```diff
@@ -22,26 +22,26 @@
   "main": "dist/rematch-loading.cjs.js",
   "module": "dist/rematch-loading.esm.js",
   "browser": "dist/rematch-loading.umd.js",
-  "types": "src/typings.d.ts",
   "scripts": {
     "clean:dist": "npx rimraf dist",
     "clean:build": "npx rimraf build",
     "clean": "npm run clean:dist & npm run clean:build",
-    "build": "NODE_ENV=production npm run clean && rollup -c"
+    "type": "tsc --emitDeclarationOnly",
+    "build": "NODE_ENV=production npm run clean && NODE_ENV=production rollup -c && npm run type"
```

Comment:
The code cannot be compressed because of an environment variable issue.

`NODE_ENV=production npm run clean` and  `NODE_ENV=production rollup -c`

@ShMcK

Reply:
This works for me as is

```
"build": "NODE_ENV=production npm run clean && rollup -c"
```

But when I run the proposed change above, it fails with warnings

```
dist/rematch-loading.umd.js → dist/rematch-loading.umd.js...
(!) `this` has been rewritten to `undefined`
https://rollupjs.org/guide/en/#error-this-is-undefined
dist/rematch-loading.umd.js
3:     typeof define === 'function' && define.amd ? define(['exports'], factory) :
4:     (global = global || self, factory(global.RematchLoading = {}));
5: }(this, function (exports) { 'use strict';
     ^
6: 
7:     /*! *****************************************************************************
```

I don't have the `code cannot be compressed because of an environment variable issue.` Could you explain more about it? It may be a non-issue.

Reply:
```
"build": "NODE_ENV=production npm run clean && rollup -c"
```
https://github.com/rematch/rematch/blob/acbfcaab8b25431c70600c517b512751c44c654d/plugins/loading/rollup.config.js#L12

`process.env.NODE_ENV ` will get an `undefined`

Reply:
@ShMcK

Reply:
```bash
▶ node -v
v12.10.0

▶ npm -v
6.11.3
```

Reply:
You''re right.

`"build": "npm run clean && NODE_ENV=production rollup -c"` should work in that case?

Reply:
@ShMcK Yes, It works.

Reply:
@ShMcK I modify the code.